localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ ursine:0 "Ursinism"
+ ursine_adj:0 "Ursine"
+ ursine_adherent:0 "Ursinist"
+ ursine_adherent_plural:0 "Ursinists"
+ ursine_desc:0 "$ursine$ is a tradition practiced by favouring balance of the wild things and the natural world. The twin ancient guardians Ursol and Ursoc receive veneration mostly from Ursol’s descendants, the furbolgs. Furbolgs attuned to nature grow into druids and shamans, following Ursol’s woven path. Ursoc's path on the other hand is saturated with blood. Here warriors and hunters learn the ferocity of nature."
+ 
+ kaldorei_religion:0 "Elunism"
+ kaldorei_religion_adj:0 "Elunic"
+ kaldorei_religion_adherent:0 "Elunist"
+ kaldorei_religion_adherent_plural:0 "Elunists"
+ kaldorei_religion_desc:0 "$kaldorei_religion_adherent_plural$ worship the pantheon of gods, led by the moon goddess Elune. Her priests, the sisters of Elune, have considerable authority in the society. On the other hand, the Elunic druids play a significant role and also have a close kinship with most of the Ancient Guardians. Blessed by their gods, the Elunists consider it their duty to protect nature at the cost of their lives and honour the sacrifices of the ancestors."
+ 
+ agamaggan_worship:0 "Agamagganism"
+ agamaggan_worship_adj:0 "Agamagganic"
+ agamaggan_worship_adherent:0 "Agamagganist"
+ agamaggan_worship_adherent_plural:0 "Agamagganists"
+ agamaggan_worship_desc:0 "Agamaggan, the great boar, was one of the ancient guardians and one of the first and most powerful beings to ever live. Shamans and geomancers that worship him call upon the strength of the great boar to help them in battle and in their rituals."
+ 
+ air_group:0 "Aerism"
+ air_group_adj:0 "Aerial"
+ air_group_adherent:0 "Aerist"
+ air_group_adherent_plural:0 "Aerists"
+ air_group_desc:0 "Among all the elements, the Aerists regard air as the main force and worship air elementals as deities."
+ 
+ alakir_worship:0 "Al'Akirism"
+ alakir_worship_adj:0 "Al'Akiric"
+ alakir_worship_adherent:0 "Al'Akirist"
+ alakir_worship_adherent_plural:0 "Al'Akirists"
+ alakir_worship_desc:0 "$alakir_worship_adherent_plural$ worship one of the Elemental Lords, Al'Akir the Windlord, as their main god. Other air elementals are seen as minor deities."
+ 
+ earth_group:0 "Geoism"
+ earth_group_adj:0 "Geo"
+ earth_group_adherent:0 "Geoist"
+ earth_group_adherent_plural:0 "Geoists"
+ earth_group_desc:0 "Among all the elements, the Geoists regard earth as the main force and worship earth elementals as deities."
+ 
+ therazane_worship:0 "Therazanism"
+ therazane_worship_adj:0 "Therazanic"
+ therazane_worship_adherent:0 "Therazanist"
+ therazane_worship_adherent_plural:0 "Therazanists"
+ therazane_worship_desc:0 "$therazane_worship_adherent_plural$ worship one of the Elemental Lords, Therazane the Stonemother, as their main god. Other earth elementals are seen as minor deities."
```

This isn't all that important, but as Therazane is referred to as female, shouldn't it be goddess? Can see arguments for and against, but wanted to point it out.
```suggestion
 therazane_worship_desc:0 "$therazane_worship_adherent_plural$ worship one of the Elemental Lords, Therazane the Stonemother, as their main goddess. Other earth elementals are seen as minor deities."
```